pxr/usdImaging/usdSkelImaging/skelRootAdapter.cpp

```diff
@@ -63,6 +63,10 @@ UsdSkelImagingSkelRootAdapter::Populate(
     if(!TF_VERIFY(prim.IsA<UsdSkelRoot>())) {
         return SdfPath();
     }
+    if(prim.IsInstance() || prim.IsInstanceProxy() || prim.IsInMaster()) {
```

Comment:
It might be simpler and faster to just check if instancerContext is non-null and bail out then.